engine/src/cmd/planet_generic.cpp

```diff
@@ -351,6 +203,8 @@ Unit* Planet::beginElement( QVector x,
                                                                QVector( 0, 0, 0 ), this, ourmat, ligh, faction, fullname, inside_out ) );
             un = p;
             p->SetOwner( this );
+            cout << "Created planet " << fullname << " of type " << p->fullname << " orbiting " << this->fullname << endl;
```

Comment:
Stray cout

Reply:
Fixed in parent branch (task_xml_refactor) and pulled here from master.